### DISCUSSION TITLE
Is it possible to set the cursor thickness?

### DISCUSSION THREAD

**Discussion by idr4n** [wezterm/wezterm#2328] [Q&A] [answered]
Hi and thanks for an amazing terminal. I found it recently and I'm using it as my daily driver.

I was wondering if there is an option for setting the cursor (beam) thickness to something wider when in insert mode in vim/neovim (I couldn't find that option in the docs), just like cursor_beam_thickness in Kitty. The current width is quite good as it is but I just like it slightly wider to make it more visible.

Thanks in advance!

**Comment by wez** [comment, marked as answer]
There isn't an option for this today. The thickness is taken from the underline thickness specified by the font that is in use.
I think that it would be a reasonable request to allow specifying a scaling factor for the underline thickness for a given font, similar to how additional attributes are passed in to `wezterm.font` (see bottom of https://wezfurlong.org/wezterm/config/lua/wezterm/font.html).

You can re-open your original question-issue to track that enhancement request if you like :-)

**Reply by idr4n** [reply]
Great, thanks for letting me know. I will reopen the question in the issues section then 👍🏻.
